www/src/pages/components/text-input/android/index.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Text Input
+description: Form inputs with sizes and style variations.
+---
+
+import { graphql } from 'gatsby';
+import { ComponentHeader } from 'components/thumbprint-components';
+
+<ComponentHeader data={props.data} />
+
+### Components
+
+There are four types of Text Input components:
+
+-   `ThumbprintTextInput` : A single line `ThumbprintEditText` with no drawables. While you can use `ThumbprintTextInputWithDrawables` without defining any drawables in place of this View, it is recommended to use this View if you do not plan on defining drawables. This is because `ThumbprintTextInputWithDrawables` has tap event listeners which will do unncessary work if no drawables are defined.
+-   `ThumbprintTextInputWithDrawables` : A single line `ThumbprintEditText` with drawables. Contains logic for capturing clicks on drawables (both start and end). Can expose drawable tap events through `.uiEvents()`.
```

Comment:
1. > Can expose drawable tap events through `.uiEvents()`.

I don't think this is true? Looking at the implementation, we don't actually expose any UIEvents. Instead, you can use `drawableStartListener` and `drawableEndListener` to listen to drawable taps.

2. Perhaps add a note saying only start & end drawables are recommended?

Reply:
Good point. I just copied the description from the `ThumbprintTextInputWithDrawables.kt` file itself. I guess the description was always misleading. I've updated it to specify the listeners.